linux_os/guide/system/software/integrity/fips/enable_fips_mode/rule.yml

```diff
@@ -3,7 +3,22 @@ documentation_complete: true
 title: Enable FIPS Mode
 
 description: |-
-{{% if product != "rhcos4" %}}
+{{% if product == "rhcos4" %}}
+    OpenShift has an installation-time flag that can enable FIPS mode
+    for the cluster. The flag <pre>fips: true</pre> must be enabled
+    at install time in the <pre>install-config.yaml</pre> file. If
+    this rule fails on an installed cluster, then this is a permanent
+    finding and cannot be fixed.
+{{% elif product == "rhel10" %}}
```

Comment:
Could we do something like `'rhel' in product and product not in ["rhel8", "rhel9"]` so we don't have to come back in RHEL 11?